<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CustomTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifiedBtn">
       <property name="text">
        <string>modified</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="text">
        <string>apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTableView</class>
   <extends>QTableView</extends>
   <header>_browser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
